socorro/processor/mozilla_transform_rules.py

```diff
@@ -868,101 +871,75 @@ def _action(self, raw_crash, raw_dumps, processed_crash, proc_meta):
 
 
 class OSPrettyVersionRule(Rule):
-    required_config = Namespace()
-    required_config.add_option(
-        'database_class',
-        doc="the class of the database",
-        default='socorro.external.postgresql.connection_context.'
-                'ConnectionContext',
-        from_string_converter=str_to_python_object,
-        reference_value_from='resource.postgresql',
-    )
-    required_config.add_option(
-        'transaction_executor_class',
-        default="socorro.database.transaction_executor."
-                "TransactionExecutorWithInfiniteBackoff",
-        doc='a class that will manage transactions',
-        from_string_converter=str_to_python_object,
-        reference_value_from='resource.postgresql',
-    )
+    '''This rule attempts to extract the most useful, singular, human
+    understandable field for operating system version. This should always be
+    attempted.
+    '''
+
+    WINDOWS_VERSIONS = {
+        '3.5': 'Windows NT',
+        '4.0': 'Windows NT',
+        '4.1': 'Windows 98',
+        '4.9': 'Windows Me',
+        '5.0': 'Windows 2000',
+        '5.1': 'Windows XP',
+        '6.0': 'Windows Vista',
+        '6.1': 'Windows 7',
+        '6.2': 'Windows 8',
+        '6.3': 'Windows 8.1',
+        '10.0': 'Windows 10',
```

Comment:
We've also got this one:

```
 Windows Server 2003  |             5 |             2
```

Otherwise this list matches what we have in -prod.